GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -395,6 +402,79 @@ extension Glia {
     }
 }
 
+extension Glia {
+    /// Deprecated, use ``configure(with configuration:theme:uiConfig:assetsBuilder:completion:)`` instead.
+    @available(*, deprecated, message: """
+            Deprecated, use ``configure(with configuration:theme:uiConfig:assetsBuilder:completion:)`` instead.
+        """
+    )
+    public func configure(
+        with configuration: Configuration,
+        uiConfig: RemoteConfiguration? = nil,
+        assetsBuilder: RemoteConfiguration.AssetsBuilder = .standard,
+        completion: @escaping (Result<Void, Error>) -> Void
+    ) throws {
+        guard environment.coreSdk.getCurrentEngagement() == nil else {
```

Comment:
1. Could we move deprecated method to Glia.Deprecated?
2. Could just call new `configure` method there?

Reply:
fixed [6aa5da3](https://github.com/salemove/ios-sdk-widgets/pull/835/commits/6aa5da3f7dc7630e7d7754440177e2a97b077103)